### ISSUE TITLE
Question about ResBlock in Generator Network

### ISSUE THREAD

**Issue by JauB1981** [DeokyunKim/Progressive-Face-Super-Resolution#9]
Hi @DeokyunKim 
Please can you tell why there is a deffirence between the paper and Model.py in ResBlock for Step =2 ?

![ProgrSR](https://user-images.githubusercontent.com/52977140/67276048-269b7380-f4c4-11e9-9eb3-f10c39a4e189.png)

In the paper, the architecture shows for step = 2 a ResBlock with 512 channel (second level), but in your Model.py we see that level 1 and 2 in ResBlock have the same channels numbers as output (wich must = 256 i think ==> dim Variable):
```

self.conv = nn.Sequential(
                            EqualConv2d(dim, dim, kernel_size=3, padding=1, stride=1),
                            nn.BatchNorm2d(dim),
                            nn.ReLU(),
                            EqualConv2d(dim, dim, kernel_size=3, padding=1, stride=1),
                            nn.BatchNorm2d(dim),
                            nn.ReLU()
                            )
```

**Comment by DeokyunKim** [comment]
Thank you for your feedback. 

There is miss typing in the paper.

The step 2 ResBlock's dimension is 256 and step 3 ResBlock's dimension is 128 as the uploaded code.

**Comment by JauB1981** [comment]
Many thanks for your quick answer :)
